client/src/components/visualisation/configMenu/TwoAxisConfigMenu.jsx

```diff
@@ -13,6 +14,32 @@ const getDatasetArray = datasetObject => {
   return datasetArray;
 };
 
+const getDatasetOptions = (datasetArray) => {
+  const output = [];
+
+  datasetArray.forEach(option => {
+    output.push({
+      value: option.id, label: option.name,
+    });
+  });
+
+  return output;
+};
+
+const getDashSelectOptionsFromColumnArray = (array) => {
+  const output = [];
+
+  if (array) {
+    array.forEach((entry, index) => {
+      output.push({
+        value: index, label: entry.title,
+      });
+    });
+  }
+
+  return output;
```

Comment:
I agree with this - but these config menus need to be refactored in any case to extract some common code. When I refactor, I will use `map` as you suggest